docs/src/pages/reference/useQuery.md

```diff
@@ -165,6 +165,9 @@ const result = useQuery({
   - Optional
   - Defaults to `true`
   - If set to `false`, structural sharing between query results will be disabled.
+- `useErrorBoundary: boolean`
+  - Defaults to the global query config's `useErrorBoundary` value, which is false
+  - Set this to true if you want mutation errors to be thrown in the render phase and propagate to the nearest error boundary
```

Comment:
Why mutation errors? This is for useQuery, isn’t it?

Reply:
My bad. That's absolutely correct. Since errors may still be thrown, I'll kill the "mutation" i.e ... 

> Set this to true if you want ~~mutation~~ errors to be thrown in the render phase and propagate to the nearest error boundary